### DISCUSSION TITLE
Convert urls in more DESCRIPTION fields (e.g. Description and Author)

### DISCUSSION THREAD

**Discussion by TimTaylor** [yihui/litedown#59] [Ideas]
It seems quite common to have linkable parts in both the Description and Author fields of the DESCRIPTION. For example:

- You often find DOIs and URLs in the Description field and I've also noticed CRAN encouraging this more on initial submissions.
- Authors often add ORCID's as a comment which can then be linked on.

Could we add these fields to those that get automatically converted at
https://github.com/yihui/litedown/blob/c78675d33cebdf744b13e16d446ffed3d48a8f37/R/package.R#L144-L146

i.e.

```R
if (i %in% c('Description', 'URL', 'BugReports', 'Author')) {
      sans_p(commonmark::markdown_html(d[[i]], extensions = 'autolink'))
    } else xfun::html_escape(d[[i]])
```

**Comment by yihui** [comment]
Yes, please feel free to submit a PR (e.g., link the name to ORCID if exists, otherwise to [URL like this](https://github.com/yihui/litedown/blob/main/site/_footer.Rmd) if exists). Thanks!

**Reply by TimTaylor** [reply]
Will do. How would you feel about attaching the ORCID to the small ORCID svg like CRAN does and only attaching URLs to the name?

**Reply by yihui** [reply]
I like this idea.
